From 389d054a82873ce22292fdc23d4214ae9a9b588c Mon Sep 17 00:00:00 2001
From: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@intel.com>
Date: Sat, 10 May 2014 11:50:19 -0700
Subject: iio: light: jsa1212: Fix read raw scale issue

Added a mising break statement in switch case of read raw function
for IIO_CHAN_INFO_SCALE.

Signed-off-by: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@intel.com>
---
 drivers/iio/light/jsa1212.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/iio/light/jsa1212.c b/drivers/iio/light/jsa1212.c
index 137ec6e..9d3c61f 100644
--- a/drivers/iio/light/jsa1212.c
+++ b/drivers/iio/light/jsa1212.c
@@ -767,6 +767,7 @@ static int jsa1212_read_raw(struct iio_dev *indio_dev,
 			ret = -EINVAL;
 			break;
 		}
+		break;
 	default:
 		ret = -EINVAL;
 		break;
-- 
1.7.9.5

